animata/text/metis-text.css

```
@@ -0,0 +1,43 @@
+  .metis-text__label {
+    display: block;
+  }
+
+  .metis-text::after {
+    content: "";
+    position: absolute;
+    left: 0;
+    top: 100%;
+    width: 100%;
+    height: 1px;
+    background: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)